### ISSUE TITLE
How to handle the homescreen backstack

### ISSUE THREAD

**Issue by greenpicat** [CMPUT301F24bubble/bubbletracks#93]
TL;DR the backstack should probably contain only a single homescreen fragment, and not all or none of them.

If it contains all of them: I click admin, then organizer. I swipe "back", expecting to exit the app, since I'm on the homescreen. Instead, it switches to the admin screen. Swiping back again switches to the entrant screen. Swiping back finally exits the app.

If none are added to the backstack: I go to the admin tab, then the profile browser, to do admin tasks. Then I swipe back to exit the profile browser, intending to switch to the facility browser. But instead I am brought back to the entrant tab. I have to select the admin tab again before I select the facility browser.
